src/main.cpp

```diff
@@ -36,6 +55,56 @@ void addGlobalCtxCleaner(graft::Manager& manager, int ms)
                 );
 }
 
+
+void startSupernodePeriodicTasks(graft::Manager& manager, size_t interval_ms)
```

Comment:
I do not think that it's the best idea keep this method in `main.cpp`, maybe better is to move it into the separated file.